assets/js/components/DashboardNavigation.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * DashboardNavigation component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { ChipSet, Chip } from '@material/react-chips';
+
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ANCHOR_ID_CONTENT,
+	ANCHOR_ID_MONETIZATION,
+	ANCHOR_ID_SPEED,
+	ANCHOR_ID_TRAFFIC,
+} from '../googlesitekit/constants';
+
+export default function DashboardNavigation() {
+	const [ selectedIds, setSelectedIds ] = useState( [] );
+
+	const handleSelect = useCallback( ( selections ) => {
+		const [ hash ] = selections;
+		if ( hash ) {
+			global.location.hash = hash;
+		}
```

Comment:
Although this works, it has poor user experience because header and navigation overlap the top part of the section when i click on a pill. I think it would be probably better if we could manually scroll to the selected area using the `window.scrollTo` function taking into account aggregated height of header and navigation elements.

cc @tofumatt and @asvinb in case you have a better idea.

Reply:
I think we will handle it in a follow up issue. Calculating the correct distance to scroll will be different after the lazy-loading stuff is implemented. Correct me if I'm wrong though, just my guess.

Reply:
Ok, then I'll defer this to @tofumatt since he should know better. Matthew, could you please confirm whether we have a dedicated task for this issue or not?

Reply:
@eugene-manuilov @kuasha420 this issue is just scaffolding the component, the navigation experience isn't intended to be final (at least it doesn't sound like that from the title and ACs 😄 ). Let's keep this as a simple working implementation for now and make sure we open a new issue for the navigation experience we want if we don't already have one for it, but anything more is out of scope here.

Reply:
@tofumatt do you know if we have a ticket for it?

Reply:
Ok, looks like we don't have a ticket for it. Created a new one: #4367. Going to mark this PR as approved.